### ISSUE TITLE
Error loading game data from path

### ISSUE THREAD

**Issue by ChocolateLoverRaj** [rstarkov/TankIconMaker#6]
```
Thread: Main

Exception: WotDataLib.WotDataException
Message: Could not parse game data for country "ussr"
  ...
  at TankIconMaker.MainWindow.ReloadData(Boolean first) in MainWindow.xaml.cs:line 399

Exception: WotDataLib.WotDataException
Message: Could not parse game data for country ""
  ...

Exception: System.InvalidOperationException
Message: Only dict values can be converted to dict.
  ...

**Comment by mjrd0815** [comment]
same issue here.
i hope a fix coming soon. ;)

**Comment by Dacogetul** [comment]
Same issue for me...

**Comment by Quaksen** [comment]
The parse error seems to happen on, at least, EU and NA clients.
However! It loads fine from the ASIA client.
Not sure what the difference between clients might be.

**Comment by RickDrain** [comment]
My first thought is that this may be related to current events and the NA server switching many (not all) "ussr" references to Ukraine.

**Comment by ChocolateLoverRaj** [comment]
> NA server switching many (not all) "ussr" references to Ukraine.

@RickDrain wdym?

**Comment by 43615** [comment]
This is getting off-topic, but such a change would be incredibly stupid:
1) The tanks are not Ukrainian.
2) Russia isn't the Soviet Union.
3) No self-respecting dev would break compatibility for the sake of virtue signalling, especially in a place where almost nobody would notice.

I believe you might be thinking of an existing mod that replaces Soviet stars in textures with Ukrainian flags, but that is obviously unrelated to TIM.

As for the actual issue: Please fix ASAP, the entire program is unusable.

**Comment by RickDrain** [comment]
No, really. On NA
Wisdom aside, it is.
I just re-loaded without mods, and if you go to tech tree the flag shown is Ukraine's. The text if you go to that page still says "USSR" but the flags on each tank are Ukraine's. 
That's what I see.

**Comment by Quaksen** [comment]
> I just re-loaded without mods, and if you go to tech tree the flag shown is Ukraine's.
Whatever you did to remove mods - didn't work. Because the mod changing flags is still in your game, if you see them.

**Comment by RickDrain** [comment]
Interesting. I'm not sure what I didn't remove. Maybe something from a mod didn't only go into res_mods. To be continued

**Comment by Quaksen** [comment]
> didn't only go into res_mods

Many mods these days go into "mods", fewer use "res_mods" these days.

**Comment by RickDrain** [comment]
Oh. Please pardon. Yes, I see. Thanks for explaining.
Is there no way to get a virgin game launch without reloading the whole game release?

**Comment by Quaksen** [comment]
Launch the game in safe mode. Also this isn't really the place for this.

Back to the topic at hand:
The program still needs an update for the newest release, it's broken on NA/EU clients.
Working on ASIA client though, at least.

**Comment by mjrd0815** [comment]
is there a new version out now or maybe a similar programm?

**Comment by ChocolateLoverRaj** [comment]
> `Exception: WotDataLib.WotDataException`

Maybe the error is actually with `WotDataLib`, and not tank icon maker. Maybe `WotDataLib` needs to be fixed or updated to a newer version.

**Comment by ChocolateLoverRaj** [comment]
> Maybe `WotDataLib` needs to be fixed or updated to a newer version.

It turns out [`WotDataLib`](https://github.com/rstarkov/WotDataLib) is a git sub-module also made by @rstarkov, so either way, we are waiting for @rstarkov for the fix.

**Comment by spkomin** [comment]
New  WotDataLib [here](http://forum.worldoftanks.eu/index.php?/topic/667058-tank-icon-maker/page__pid__18950463#entry18950463)

**Comment by mjrd0815** [comment]
> 

Thankk you! its working!

**Comment by ChocolateLoverRaj** [comment]
> Thankk you! its working!
@mjrd0815 how to make it work?

**Comment by Quaksen** [comment]
> @mjrd0815 how to make it work?

Download the linked WotDataLib from spkomin, extract the file, use those two files to replace the two already in Tank Icon Maker folder. Done.

**Comment by mjrd0815** [comment]
@ChocolateLoverRaj exactly like @Quaksen have writen it... 
sorry, was away for a while :)

**Comment by ChocolateLoverRaj** [comment]
> 

I think those files are from the pull request to wot data lib: https://github.com/rstarkov/WotDataLib/pull/2

**Comment by spkomin** [comment]
yes it is from [Fix: data loads (WoT 1.16.1.0)](https://github.com/ktulho/WotDataLib/commit/e8050631e6a4b3fa625c7e7ed1ef1f9ee6b3af8c)

**Comment by AstaRom** [comment]
Fixed in latest release version:

- https://github.com/rstarkov/TankIconMaker/releases/tag/release-v065
